variants/backend-base/config/secrets.yml

```diff
@@ -0,0 +1,26 @@
+# Do NOT put secrets directly into this file. All secrets should be loaded from ENV!
+# Be sure to restart your server when you modify this file.
+
+default: &default
+  # Your secret key is used for verifying the integrity of signed cookies.
+  # If you change this key, all old signed cookies will become invalid!
+  # Make sure the secret is at least 30 characters and all random,
+  # no regular words or you'll be exposed to dictionary attacks.
+  # You can use `rails secret` to generate a secure secret key.
+  secret_key_base: "<%= ENV.fetch('RAILS_SECRET_KEY_BASE') %>"
+
+  active_record_encryption_primary_key: "<%= ENV.fetch('ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY') %>"
+  active_record_encryption_deterministic_key: "<%= ENV.fetch('ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY') %>"
+  active_record_encryption_key_derivation_salt: "<%= ENV.fetch('ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT') %>"
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+staging:
+  <<: *default
```

Comment:
🐘 ideally we should actually only add this if a `capistrano` variant is enabled, because Heroku doesn't do `staging` so it isn't reasonable to assume all apps will have it

Reply:
I think we should keep as is because:

1. Heroku is the odd one out in not supporting a staging env out of the box - every other way of deploying Rails uses one AFAIK (not just capistrano based deploys). 
1. Having this here doesn't harm a Heroku deploy.
2. We don't currently have a template config variable for "deploying to heroku". I don't think this use-case is enough to add that complexity.